lib/run/index.js

```diff
@@ -158,16 +158,19 @@ module.exports = function (options, callback) {
                         summary: emitter.summary
                     });
 
-                    _.forEach(['environment', 'globals', 'collection'], function (item) {
+                    _.forEach(['environment', 'globals'], function (item) {
```

Comment:
Please just revert sections I don't want diff where it shouldn't be. For example, I see that collection export support is lost by mistake.